server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -2692,7 +2692,7 @@ public Snapshot allocSnapshot(Long volumeId, Long policyId, String snapshotName,
             throw new InvalidParameterValueException("VolumeId: " + volumeId + " please attach this volume to a VM before create snapshot for it");
         }
 
-        return snapshotMgr.allocSnapshot(volumeId, policyId, snapshotName, locationType);
+        return snapshotMgr.allocSnapshot(volumeId, policyId, snapshotName, locationType, null);
```

Comment:
@ravening null here means true or false ? if false, you can keep the old method.

```suggestion
        return snapshotMgr.allocSnapshot(volumeId, policyId, snapshotName, locationType, null);
```